todolist/file_store.go

```diff
@@ -13,22 +14,33 @@ type FileStore struct {
 }
 
 func NewFileStore() *FileStore {
-	return &FileStore{FileLocation: ".todos.json", Loaded: false}
+	localrepo := ".todos.json"
+	usr, _ := user.Current()
+	homerepo := fmt.Sprintf("%s/.todos.json", usr.HomeDir)
+	_, err1 := os.Stat(localrepo)
+	_, err2 := os.Stat(homerepo)
+
+	if err1 == nil {
+		return &FileStore{FileLocation: localrepo, Loaded: false}
+	} else if err2 == nil {
+		return &FileStore{FileLocation: homerepo, Loaded: false}
+	} else {
+		return &FileStore{FileLocation: "", Loaded: false}
```

Comment:
I think it makes sense to exit the app here, instead of returning a blank `FileLocation`.  That way, you don't have to do the checks above for a nil `App`, which I think feels a little weird.  If I request an instance of `App`, I should expect an instance of `App`.